src/TelemetryChannels/ServerTelemetryChannel/Shared/SamplingTelemetryProcessor.cs

```diff
@@ -65,6 +66,7 @@ public void Process(ITelemetry item)
                         TelemetryChannelEventSource.Log.ItemSampledOut(item.ToString());
                     }
 
+                    TelemetryDebugWriter.WriteTelemetry(item, this.GetType().FullName);
```

Comment:
Does `this.GetType().FullName` includes namespaces? I'd say we only need name to make output more compact